### DISCUSSION TITLE
貌似部分地区运营商开始屏蔽Cloudflare

### DISCUSSION THREAD

**Discussion by wzwtt** [XIU2/CloudflareSpeedTest#324] [讨论]
之前一直通过优选出的ip正常使用，但从大约上个星期开始，经常出现间歇性的无法筛选出任何ip的情况，此时会发现无法打开任何套用cloudflare cdn加速的网站，也无法打开cloudflare官网，ping cloudflare.com显示超时。在偶尔能筛选出ip时，在筛选完后立即ping所筛选出的ip，发现丢包严重，没过多久后该ip又超时了。
我本地电脑分别接入辽宁联通宽带ipv4、辽宁省教育网ipv4、辽宁电信蜂窝网络ipv4和ipv6、浙江电信宽带ipv4和ipv6、浙江电信蜂窝网络ipv4和ipv6、浙江移动宽带ipv4、浙江移动蜂窝网络ipv4和ipv6后多次测试，发现在电信网络和浙江移动蜂窝网络ipv6一切正常，该现象发生在辽宁联通宽带ipv4、辽宁省教育网ipv4、浙江移动宽带ipv4、浙江移动蜂窝网络ipv4上。
不知其他地区其他运营商的结果如何？

**Comment by XIU2** [comment]
你说的这个情况已经有其他类似 Issues 了，本质上都是同一种问题，请集中在一起讨论，勿多开。
#188 #217

**Comment by fredericky123** [comment]
近几天遇到了一样的情况 cloudfare被彻底阻断了
下一步怎么办 有什么解决办法吗

**Comment by XIU2** [comment]
@fredericky123 如果真是墙/运营商干的，那么就没有解决办法，毕竟你作为用户无法反向钳制上层路由。

**Comment by fredericky123** [comment]
万一cdn的方案要被淘汰了，那下一步有什么新兴的协议或者方案用于party吗 求推荐

**Reply by xiaorong61** [reply]
hysteria2 / vless reality
